src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -24,6 +25,10 @@ const PortraitVideoCarousel = ({
     null,
   );
 
+  const { isLite } = useContext(RequestContext);
```

Comment:
We've been using `use` instead of `useContext` - see https://github.com/bbc/simorgh/pull/12637
```suggestion
  const { isLite } = use(RequestContext);
```